Lib/test/test_generators.py

```diff
@@ -2247,6 +2247,11 @@ def printsolution(self, x):
   ...
 SyntaxError: 'yield' outside function
 
+>>> yield from [1,2]
```

Comment:
Maybe add a test with lambda as well since they are "almost like" functions (but not from a compiler PoV).

Reply:
I'll do it in a seperate PR

Reply:
I added another test as requested in https://github.com/python/cpython/pull/121722